docs/application/native/guides/ui/dali/ui-components-n.md

```diff
@@ -24,6 +24,7 @@ The following table lists the available UI components.
 | [ImageView](./imageview-n.md)         | An image view that renders an image.     | `ImageView`                              |
 | [ItemView](./itemview-n.md)           | An item view that renders item sets in a scrollable layout. | `ItemView`, `ItemFactory`, `ItemLayout`, `Scrollable` |
 | [Model3dView](./model3dview-n.md)     | A model view that displays static 3D content. | `Model3dView`                            |
+| [ProgressBar](./progressbar-n.md)     | A control to give the user an indication of the progress of an operation. | `ProgressBar`                              |
 | [ScrollView](./scrollview-n.md)       | A scroll view to provide scrollable view. | `ScrollView`, `Scrollable`, `ScrollViewEffect`, `ScrollViewPagePathEffect` |
 | [Slider](./slider-n.md)               | A control that indicates a modifiable value within a specific range. | `Slider`                                 |
 | [TableView](./tableview-n.md)         | A table view that can align child actors in a grid like layout. | `TableView`                              |
```

Comment:
Modify the paragraph after the table as per the suggestion below.

The base class for the components is Dali::Toolkit::Control (in [mobile] (http://org.tizen.native.mobile.apireference/classDali_1_1Toolkit_1_1Control.html) and [wearable] (http://org.tizen.native.wearable.apireference/classDali_1_1Toolkit_1_1Control.html) applications). It can also be used to create your own custom UI components. 

For more information control class, see [Control](control-base.md). You can [customize the look of the UI components with stylesheets](styling.md). For a reusable rendering logic that can be used by all UI components, take advantage of [DALi visuals](visuals.md).